composer.json

```diff
@@ -24,9 +24,9 @@
 	"require": {
 		"php": ">= 7.4.0, <= 8.3.99",
 		"ext-json": "*",
-		"typo3/cms-core": "^11.5",
-		"typo3/cms-frontend": "^11.5",
-		"typo3/cms-felogin": "^11.5",
+		"typo3/cms-core": "^11.5 | ^12.4",
```

Comment:
According to composer specs `||` is the way to go:
https://getcomposer.org/doc/articles/versions.md#version-range

Reply:
is changed